docs/pages/build/advanced-credentials-configuration.md

```diff
@@ -33,9 +33,24 @@ The `credentials.json` file should be located at the root of your project, next
 It specifies credentials for Android and iOS (but you can configure only one of these platforms if you don't want to build for the other).
 **Remember to add `credentials.json` to `.gitignore` so you won't accidentally commit it to the repository (and therefore leak your secrets)**.
 
-## Android Credentials
+## Configuring credentials.json
 
-### Obtaining the Keystore
+You can either let Expo generate new credentials and store them on Expo servers or create them manually and put them in credentials.json. Whichever option you choose you can always sync credentials one way or the other with `expo eas:credentials:sync`.
+
+
+## Automatic configuration
+
+If you already have credentials configured for an app with the same `slug`, `owner` (if you are using teams), and `ios.bundleIdentifier` (for iOS only) you can generate the `credentials.json` file from credentials that are stored on Expo servers. To generate (or update) that file run `expo eas:credentials:sync` and in the first prompt select option to update `credentials.json` and in the second one select for which platforms you want to update.
```

Comment:
```suggestion
If you already have credentials configured for an app with the same `slug`, `owner` (if you are using teams), and `ios.bundleIdentifier` (for iOS only), you can generate the `credentials.json` file from those credentials that are stored on Expo's servers. To generate (or update) that file, run `expo eas:credentials:sync`. In the first prompt, select the option to update `credentials.json`, then in the second one, select which platform(s) you wish to update.
```

does android.package not matter in this case?

Reply:
www credential are using the same logic as for v1 builds so
- android app is identified by owner and slug 
- ios app is identified by owner, slug and bundleIdentifier

New credentials logic that @quinlanj is working on might change that but for now, it's this way